index.d.ts

```diff
@@ -1,5 +1,9 @@
-declare function anySignal(signals: AbortSignal[]): AbortSignal;
+interface CompositeSignal extends AbortSignal {
+    clear(): void
+}
 
-export {anySignal};
+declare function anySignal(signals: Array<AbortSignal | undefined | null>): AbortSignal;
```

Comment:
The return type needs to be updated to the new signal type:

```suggestion
declare function anySignal(signals: Array<AbortSignal | undefined | null>): CompositeSignal;
```

or:

```suggestion
declare function anySignal(signals: Array<AbortSignal | undefined | null>): ClearableSignal;
```

Reply:
Clearable it is @achingbrain